### DISCUSSION TITLE
How to Change MPV path

### DISCUSSION THREAD

**Discussion by ccrsxx** [pystardust/ani-cli#546] [Q&A] [answered]
I have MPV installed already with all of my settings. I want to use my current MPV instead of the stock one that scoop installed. I tried looking at the source code but can't find it. Is there a way to change the MPV path?

**Comment by RaynardGerraldo** [comment, marked as answer]
> I have MPV installed already with all of my settings. I want to use my current MPV instead of the stock one that scoop installed. I tried looking at the source code but can't find it. Is there a way to change the MPV path?

Without the scoop installed mpv, can you run it from the command line? If so then you can uninstall scoop's mpv and use yours,it will work

And if it doesnt run from the command line,you can try and change https://github.com/pystardust/ani-cli/blob/a16d24d0bcc517e6a5c36cfc7daa55860009c4c3/ani-cli#L463  to your mpv path

**Reply by ccrsxx** [reply]
Ok, it works now.

I need to include the full path of the MPV to make it work. It's strange because I already have MPV set in the path, but it didn't work unless I specify its full path manually in the source code.

Anyway, thanks for the help!

**Reply by RaynardGerraldo** [reply]
No prob!
